currency/storage.go

```diff
@@ -248,36 +265,31 @@ func (s *Storage) ForeignExchangeUpdater() {
 
 // SeedCurrencyAnalysisData sets a new instance of a coinmarketcap data.
 func (s *Storage) SeedCurrencyAnalysisData() error {
-	b, err := ioutil.ReadFile(s.path)
-	if err != nil {
-		err = s.FetchCurrencyAnalysisData()
+	var firstRun bool
+	if s.currencyCodes.LastMainUpdate.IsZero() {
+		firstRun = true
+		b, err := ioutil.ReadFile(s.path)
 		if err != nil {
-			return s.WriteCurrencyDataToFile(s.path, false)
+			return s.FetchCurrencyAnalysisData()
+		}
+		var file *File
```

Comment:
rename to something that doesn't collide with `file` package

Reply:
🤦 yes.